2021-08-10 22:57:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS TBL_TASK  
  ID[*] UUID DEFAULT UUID(),
  NAME VARCHAR(50) NOT NULL,
  DONE BOOLEAN NOT NULL DEFAULT FALSE,
  TASK_CREATED TIMESTAMP NOT NULL DEFAULT NOW(),
  CONSTRAINT TASK_PK PRIMARY KEY(ID)
 "; SQL statement:
create table if not exists tbl_task  
  id UUID default uuid(),
  name varchar(50) not null,
  done boolean not null default false,
  task_created timestamp not null default now(),
  constraint task_pk primary key(id)
  [42000-200]
2021-08-10 23:03:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (WAKE UP)
"; expected "identifier"; SQL statement:
insert into table tbl_task (Wake up)
 [42001-200]
2021-08-10 23:03:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (SHOWER)
"; expected "identifier"; SQL statement:
insert into table tbl_task (Shower)
 [42001-200]
2021-08-10 23:03:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (EAT BREAKFAST)
"; expected "identifier"; SQL statement:
insert into table tbl_task (Eat Breakfast)
 [42001-200]
2021-08-10 23:03:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (NAME) VALUE ('Wake up')
"; expected "identifier"; SQL statement:
insert into table tbl_task (name) value ('Wake up')
 [42001-200]
2021-08-10 23:03:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (NAME) VALUE ('Shower')
"; expected "identifier"; SQL statement:
insert into table tbl_task (name) value ('Shower')
 [42001-200]
2021-08-10 23:03:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (NAME) VALUE ('Eat Breakfast')
"; expected "identifier"; SQL statement:
insert into table tbl_task (name) value ('Eat Breakfast')
 [42001-200]
2021-08-10 23:04:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (NAME) VALUE ('Wake up');"; expected "identifier"; SQL statement:
insert into table tbl_task (name) value ('Wake up'); [42001-200]
2021-08-10 23:04:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (NAME) VALUE ('Shower');"; expected "identifier"; SQL statement:
insert into table tbl_task (name) value ('Shower'); [42001-200]
2021-08-10 23:04:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (NAME) VALUE ('Eat Breakfast');"; expected "identifier"; SQL statement:
insert into table tbl_task (name) value ('Eat Breakfast'); [42001-200]
2021-08-10 23:12:58 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.populateClientAppName(GenericDataSource.java:175)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.openConnection(GenericDataSource.java:138)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.openConnection(H2DataSource.java:74)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:100)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:59)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:109)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.<init>(GenericDataSource.java:107)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.<init>(H2DataSource.java:43)
	at org.jkiss.dbeaver.ext.h2.model.H2MetaModel.createDataSourceImpl(H2MetaModel.java:47)
	at org.jkiss.dbeaver.ext.generic.GenericDataSourceProvider.openDataSource(GenericDataSourceProvider.java:106)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:919)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:70)
	at org.jkiss.dbeaver.runtime.jobs.ConnectionTestJob.run(ConnectionTestJob.java:103)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2021-08-10 23:13:04 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.populateClientAppName(GenericDataSource.java:175)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.openConnection(GenericDataSource.java:138)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.openConnection(H2DataSource.java:74)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:100)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:59)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:109)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.<init>(GenericDataSource.java:107)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.<init>(H2DataSource.java:43)
	at org.jkiss.dbeaver.ext.h2.model.H2MetaModel.createDataSourceImpl(H2MetaModel.java:47)
	at org.jkiss.dbeaver.ext.generic.GenericDataSourceProvider.openDataSource(GenericDataSourceProvider.java:106)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:919)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:70)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.runSync(ConnectJob.java:98)
	at org.jkiss.dbeaver.ui.actions.datasource.DataSourceHandler.connectToDataSource(DataSourceHandler.java:113)
	at org.jkiss.dbeaver.ui.actions.datasource.UIServiceConnectionsImpl.initConnection(UIServiceConnectionsImpl.java:70)
	at org.jkiss.dbeaver.model.navigator.DBNDataSource.initializeNode(DBNDataSource.java:158)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:223)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.navigator.DBNUtils.getNodeChildrenFiltered(DBNUtils.java:78)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:49)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:1)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:88)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:72)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2021-08-10 23:13:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS TBL_TASK  
  ID[*] UUID DEFAULT RANDOM_UUID(),
  NAME VARCHAR(50) NOT NULL,
  DONE BOOLEAN NOT NULL DEFAULT FALSE,
  TASK_CREATED TIMESTAMP NOT NULL DEFAULT NOW(),
  CONSTRAINT TASK_PK PRIMARY KEY(ID),
  CONSTRAINT UNIQ_NAME UNIQUE NAME
 "; SQL statement:
create table if not exists tbl_task  
  id UUID default random_uuid(),
  name varchar(50) not null,
  done boolean not null default false,
  task_created timestamp not null default now(),
  constraint task_pk primary key(id),
  constraint uniq_name unique name
  [42000-200]
2021-08-10 23:14:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS TBL_TASK  
  ID[*] UUID DEFAULT RANDOM_UUID(),
  NAME VARCHAR(50) NOT NULL,
  DONE BOOLEAN NOT NULL DEFAULT FALSE,
  TASK_CREATED TIMESTAMP NOT NULL DEFAULT NOW(),
  CONSTRAINT TASK_PK PRIMARY KEY(ID),
  CONSTRAINT UNIQ_NAME UNIQUE (NAME)
 "; SQL statement:
create table if not exists tbl_task  
  id UUID default random_uuid(),
  name varchar(50) not null,
  done boolean not null default false,
  task_created timestamp not null default now(),
  constraint task_pk primary key(id),
  constraint uniq_name unique (name)
  [42000-200]
2021-08-10 23:14:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE TBL_TASK IF[*] NOT EXISTS   
  ID UUID DEFAULT RANDOM_UUID(),
  NAME VARCHAR(50) NOT NULL,
  DONE BOOLEAN NOT NULL DEFAULT FALSE,
  TASK_CREATED TIMESTAMP NOT NULL DEFAULT NOW(),
  CONSTRAINT TASK_PK PRIMARY KEY(ID),
  CONSTRAINT UNIQ_NAME UNIQUE (NAME)
 "; SQL statement:
create table tbl_task if not exists   
  id UUID default random_uuid(),
  name varchar(50) not null,
  done boolean not null default false,
  task_created timestamp not null default now(),
  constraint task_pk primary key(id),
  constraint uniq_name unique (name)
  [42000-200]
2021-08-10 23:14:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE TBL_TASK IF[*] NOT EXISTS   
  ID UUID DEFAULT RANDOM_UUID(),
  NAME VARCHAR(50) NOT NULL,
  DONE BOOLEAN NOT NULL DEFAULT FALSE,
  TASK_CREATED TIMESTAMP NOT NULL DEFAULT NOW(),
  CONSTRAINT TASK_PK PRIMARY KEY(ID),
  CONSTRAINT UNIQ_NAME UNIQUE (NAME)
 "; SQL statement:
create table tbl_task if not exists   
  id UUID default random_uuid(),
  name varchar(50) not null,
  done boolean not null default false,
  task_created timestamp not null default now(),
  constraint task_pk primary key(id),
  constraint uniq_name unique (name)
  [42000-200]
2021-08-10 23:15:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "TBL_TASK[*]"; expected "TRUNCATE"; SQL statement:
tbl_task [42001-200]
2021-08-10 23:15:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS TBL_TASK  
  ID[*] UUID DEFAULT RANDOM_UUID(),
  NAME VARCHAR(50) NOT NULL,
  DONE BOOLEAN NOT NULL DEFAULT FALSE,
  TASK_CREATED TIMESTAMP NOT NULL DEFAULT NOW(),
  CONSTRAINT TASK_PK PRIMARY KEY(ID),
  CONSTRAINT UNIQ_NAME UNIQUE (NAME)
 "; SQL statement:
create table if not exists tbl_task  
  id UUID default random_uuid(),
  name varchar(50) not null,
  done boolean not null default false,
  task_created timestamp not null default now(),
  constraint task_pk primary key(id),
  constraint uniq_name unique (name)
  [42000-200]
2021-08-10 23:17:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (NAME) VALUE ('Wake up');"; expected "identifier"; SQL statement:
insert into table tbl_task (name) value ('Wake up'); [42001-200]
2021-08-10 23:17:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (NAME) VALUE ('Shower');"; expected "identifier"; SQL statement:
insert into table tbl_task (name) value ('Shower'); [42001-200]
2021-08-10 23:17:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (NAME) VALUE ('Eat Breakfast');"; expected "identifier"; SQL statement:
insert into table tbl_task (name) value ('Eat Breakfast'); [42001-200]
2021-08-10 23:17:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (NAME) VALUES ('Wake up');"; expected "identifier"; SQL statement:
insert into table tbl_task (name) values ('Wake up'); [42001-200]
2021-08-10 23:17:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (NAME) VALUES ('Shower');"; expected "identifier"; SQL statement:
insert into table tbl_task (name) values ('Shower'); [42001-200]
2021-08-10 23:17:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (NAME) VALUES ('Eat Breakfast');"; expected "identifier"; SQL statement:
insert into table tbl_task (name) values ('Eat Breakfast'); [42001-200]
2021-08-10 23:18:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (NAME) VALUES ('Wake up')"; expected "identifier"; SQL statement:
insert into table tbl_task (name) values ('Wake up') [42001-200]
2021-08-10 23:18:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (NAME) VALUES ('Shower')"; expected "identifier"; SQL statement:
insert into table tbl_task (name) values ('Shower') [42001-200]
2021-08-10 23:18:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO TABLE[*] TBL_TASK (NAME) VALUES ('Eat Breakfast')"; expected "identifier"; SQL statement:
insert into table tbl_task (name) values ('Eat Breakfast') [42001-200]
2021-08-10 23:44:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "0" for parameter "columnIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcResultSetMetaData.checkColumnIndex(JdbcResultSetMetaData.java:447)
	at org.h2.jdbc.JdbcResultSetMetaData.getColumnName(JdbcResultSetMetaData.java:85)
	at works.hop.jdbc.s_1_select_basic.Select.select(Select.java:32)
	at works.hop.jdbc.s_1_select_basic.Select.main(Select.java:12)
2021-08-10 23:45:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "0" for parameter "columnIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcResultSetMetaData.checkColumnIndex(JdbcResultSetMetaData.java:447)
	at org.h2.jdbc.JdbcResultSetMetaData.getColumnName(JdbcResultSetMetaData.java:85)
	at works.hop.jdbc.s_1_select_basic.Select.select(Select.java:32)
	at works.hop.jdbc.s_1_select_basic.Select.main(Select.java:12)
2021-08-10 23:46:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "0" for parameter "columnIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcResultSetMetaData.checkColumnIndex(JdbcResultSetMetaData.java:447)
	at org.h2.jdbc.JdbcResultSetMetaData.getColumnName(JdbcResultSetMetaData.java:85)
	at works.hop.jdbc.s_1_select_basic.Select.select(Select.java:32)
	at works.hop.jdbc.s_1_select_basic.Select.main(Select.java:12)
2021-08-10 23:48:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "0" for parameter "columnIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:3212)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3251)
	at org.h2.jdbc.JdbcResultSet.getObject(JdbcResultSet.java:3827)
	at works.hop.jdbc.s_1_select_basic.Select.select(Select.java:34)
	at works.hop.jdbc.s_1_select_basic.Select.main(Select.java:12)
2021-08-10 23:49:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "0" for parameter "columnIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:3212)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3251)
	at org.h2.jdbc.JdbcResultSet.getObject(JdbcResultSet.java:3827)
	at works.hop.jdbc.s_1_select_basic.Select.select(Select.java:34)
	at works.hop.jdbc.s_1_select_basic.Select.main(Select.java:12)
2021-08-11 00:18:24 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.populateClientAppName(GenericDataSource.java:175)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.openConnection(GenericDataSource.java:138)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.openConnection(H2DataSource.java:74)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:100)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:59)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:109)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.<init>(GenericDataSource.java:107)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.<init>(H2DataSource.java:43)
	at org.jkiss.dbeaver.ext.h2.model.H2MetaModel.createDataSourceImpl(H2MetaModel.java:47)
	at org.jkiss.dbeaver.ext.generic.GenericDataSourceProvider.openDataSource(GenericDataSourceProvider.java:106)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:919)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:70)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2021-08-11 00:19:00 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.populateClientAppName(GenericDataSource.java:175)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.openConnection(GenericDataSource.java:138)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.openConnection(H2DataSource.java:74)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:100)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:59)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:109)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.<init>(GenericDataSource.java:107)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.<init>(H2DataSource.java:43)
	at org.jkiss.dbeaver.ext.h2.model.H2MetaModel.createDataSourceImpl(H2MetaModel.java:47)
	at org.jkiss.dbeaver.ext.generic.GenericDataSourceProvider.openDataSource(GenericDataSourceProvider.java:106)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:919)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:70)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
